server/src/controllers/login.js

```diff
@@ -0,0 +1,44 @@
+const Boom = require('@hapi/boom');
+const bcrypt = require('bcrypt');
+require('env2')('./config.env');
+
+const { checkEmail } = require('../database/queries');
+const { sendCookies } = require('../utils/jwt');
+const schema = require('./validation/loginSchema');
+
+const login = (req, res, next) => {
+  const { email, password } = req.body;
+  schema
+    .validateAsync({ email, password }, { abortEarly: false })
+    .then(() => checkEmail(email))
+    .then(({ rows }) => {
+      if (rows.length > 0) {
+        return rows[0];
+      }
+      return next(Boom.unauthorized('You need to signup first'));
+    })
+    .then((userData) => {
+      if (userData.is_active) {
+        req.userID = userData.id;
+        req.isAdmin = userData.is_Admin;
```

Comment:
instead of doing this, which is not the best practice, since you are mutating on the request object, go for immutability, create a variable object called tokenData at the top of your code add these variables to it and then just spread it when you 'sign' and ready the token to get sent